replica/memtable.cc

```diff
@@ -31,30 +31,26 @@ static mutation_reader make_partition_snapshot_flat_reader_from_snp_schema(
         std::any pointer_to_container,
         streamed_mutation::forwarding fwd, memtable& memtable);
 
-void memtable::memtable_encoding_stats_collector::update_timestamp(api::timestamp_type ts) noexcept {
-    if (ts != api::missing_timestamp) {
-        encoding_stats_collector::update_timestamp(ts);
-        min_max_timestamp.update(ts);
-    }
-}
-
 memtable::memtable_encoding_stats_collector::memtable_encoding_stats_collector() noexcept
     : min_max_timestamp(0, 0)
+    , min_live_timestamp(api::max_timestamp)
+    , min_live_row_marker_timestamp(api::max_timestamp)
 {}
 
 void memtable::memtable_encoding_stats_collector::update(atomic_cell_view cell) noexcept {
-    update_timestamp(cell.timestamp());
+    auto is_live = ::is_live(cell.is_live());
```

Comment:
I'm not sure it will compile, since ::is_live is a bool class which is a strong type with `constexpr explicit bool_class(bool v) noexcept : _value(v) { }`

Reply:
this will need to be written as:
```c++
::is_live is_live = ::is_live(cell.is_live());
```

Reply:
Or is_live is_live{...}.